magenta/models/melody_rnn/melody_rnn_sequence_generator.py

```diff
@@ -192,24 +196,30 @@ def generate_melody(self, num_steps, primer_melody):
         encoder_decoder.max_note,
         encoder_decoder.transpose_to_key)
 
-    inputs = self._session.graph.get_collection('inputs')[0]
-    initial_state = self._session.graph.get_collection('initial_state')[0]
-    final_state = self._session.graph.get_collection('final_state')[0]
-    softmax = self._session.graph.get_collection('softmax')[0]
+    graph_inputs = self._session.graph.get_collection('inputs')[0]
+    graph_initial_state = self._session.graph.get_collection('initial_state')[0]
+    graph_final_state = self._session.graph.get_collection('final_state')[0]
+    graph_softmax = self._session.graph.get_collection('softmax')[0]
+    graph_temperature = self._session.graph.get_collection('temperature')
```

Comment:
We can't because in old bundles, that collection won't exist. We could require it, but it would break backwards compatibility with old graphs.